code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -7,24 +7,24 @@
 	buckle_lying = FALSE
 	mob_biotypes = list(MOB_ORGANIC, MOB_HUMANOID)
 	blocks_emissive = EMISSIVE_BLOCK_UNIQUE
-	//Hair colour and style
+	///Hair colour
 	var/hair_color = "000"
+	///Hair style
 	var/hair_style = "Bald"
-
-	//Facial hair colour and style
+	///Colour used for the hair gradient.
+	var/gradient_color = "000"
+	///Style used for the hair gradient.
+	var/gradient_style
```

Comment:
Why do they need to be defined on both humans and species?

Reply:
Moved it off